src/Command/RecipesCommand.php

```diff
@@ -96,35 +98,49 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 0;
         }
 
-        // display a resume of all packages
-        $write = [
-            '',
-            '<bg=blue;fg=white>                      </>',
-            '<bg=blue;fg=white> Available recipes.   </>',
-            '<bg=blue;fg=white>                      </>',
-            '',
-        ];
+        $outdated = $input->getOption('outdated');
 
+        $write = [];
         /** @var Recipe $recipe */
         foreach ($recipes as $name => $recipe) {
             $lockRef = $this->symfonyLock->get($name)['recipe']['ref'] ?? null;
 
-            $additional = '';
+            $additional = null;
             if (null === $lockRef && null !== $recipe->getRef()) {
                 $additional = '<comment>(recipe not installed)</comment>';
             } elseif ($recipe->getRef() !== $lockRef) {
                 $additional = '<comment>(update available)</comment>';
             }
-            $write[] = sprintf(' * %s %s', $name, $additional);
+
+            if (!$outdated) {
+                $write[] = sprintf(' * %s %s', $name, $additional);
+
+                continue;
+            }
+
+            if (null !== $additional) {
+                $write[] = sprintf(' * %s %s', $name, $additional);
+            }
```

Comment:
This all may be correct, but it doesn’t read clearly to me. I expect the algorithm to read like this: 

* determine if the recipe is outdated (you already do this first via the $additional variable)
* then, if the recipe is NOT outdated AND the outdated flag is set, continue (skip printing)
* else, print (because the recipe is either outdated or we are showing all recipes)